workflows/VGP-assembly-v2/Pre-Curation_PretextMap/PretextMap_Generation.ga

```diff
@@ -0,0 +1,2048 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
```

Comment:
This is what users see in the workflow list and the run form, can you put the first sentence from the readme here (this workflow does xyz ...)